README.md

```diff
@@ -231,12 +231,12 @@ Examples to start the fictitious file named `foo_measurement.py`:
 
 ### Create Executable for Python Scripts
 
-To create an executable from a measurement, measurement authors can use the [pyinstaller](https://www.pyinstaller.org/) tooling. During the executable creation, the user can also embed the User Interface file using the `--add-data "<path_of_the_UI_File>;."`.
+To create an executable from a measurement, you can use the [pyinstaller](https://www.pyinstaller.org/) tooling. If you are using a virtual environment, install `pyinstaller` to your venv.
 
-Typical Pyinstaller command to build executable.
+Typical PyInstaller command to build executable:
 
 ```cmd
-pyinstaller --onefile --console --add-data "<path_of_the_UI_File>;." --paths .venv\Lib\site-packages\ <path_of_the_measurement_script>
+pyinstaller --onefile --console --paths .venv\Lib\site-packages\ <path_of_the_measurement_script>
```

Comment:
The measurement script is typically named `measurement.py` and is typically located in the project root, so let's provide an example path.

Is the trailing backslash after site-packages necessary? It makes it look like there is an extra space (like the classic `rm -fr /usr /local/bin/foo` bug).

```suggestion
pyinstaller --onefile --console --paths .venv\Lib\site-packages measurement.py
```

Reply:
Just tested this out. Looks like the trailing backslash is unnecessary. I will remove it.